src/main/java/com/manardenza/service/UserService.java

```diff
@@ -0,0 +1,36 @@
+package com.manardenza.service;
+
+import com.manardenza.dao.UserDaoImpl;
+import com.manardenza.entity.User;
+import com.manardenza.login.CurrentUser;
+
+public class UserService {
+
+    private static UserService instance;
+    private UserDaoImpl userDao;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "userDao" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)